projects/degenerative/index.js

```diff
@@ -13,6 +13,7 @@ async function tvl(api) {
 }
 
 module.exports = {
+  doublecounted: true,
```

Comment:
can you add a comment why it is marked as doublecounted? it is not very clear, looks like we are counting funds in an multisig?